### ISSUE TITLE
PP displayer for 100% is inconsistent

### ISSUE THREAD

**Issue by Ztumpie905509** [KedamaOvO/RealTimePPDisplayer#27]
The PP displayed between the choose-song menu and final results is inconsistent.

Take [Now Loading!!!! [Extra] by doormat](https://osu.ppy.sh/beatmapsets/480609#osu/1027065) as an example,

The pp showed in the Choose-song menu for 100% is 252.90pp
![screenshot085](https://user-images.githubusercontent.com/48085506/54065395-6d04d080-425b-11e9-955f-bd24000fd270.png)

However, when I use Auto mod to spectate the map, the bot scored it 100% but the pp showed is 251.11pp instead of 252.90pp
![screenshot084](https://user-images.githubusercontent.com/48085506/54065404-a3425000-425b-11e9-9fcf-d22926801586.png)

May I ask which value (251.11pp or 252.90pp) is more accurate ?
Maybe also fix the inconsistent problem ?

**Comment by KedamaOvO** [comment]
Make sure "Listen interval" is a suitable value.
![](https://i.imgur.com/KHORWHI.png)

If it is too long, RTPPD may not be able to calculate the last pp.
(pp calculation only works in the Playing state)

```
                         Timeline
+------------------------------+--------------------------+
|               Playing        |          Rank            |
|                              |                          |
|            T1    T2    T3  T4|                          |
|            |     |     |   | |                          |
| OSU      +-+-----+-----+---+----------------------+     |
|              PP1   PP2   PP3 |  Final PP(No calculation)|
|              |     |     |   |  |                       |
| RTPPD    +---------------+------+-----------------+     |
|              ^     ^         |                          |
|              +--+--+         |                          |
|                 |         Only working at Playing       |
|            Listen Interval                              |
+---------------------------------------------------------+
```

**Comment by Ztumpie905509** [comment]
I think this is a default value as i never touched it.

![Annotation 2019-03-09 175456](https://user-images.githubusercontent.com/48085506/54069597-7cecd680-4294-11e9-9324-318f4961b621.jpg)

For the length of the song, it is just 01:27, i don't think this is too long.

**Comment by KedamaOvO** [comment]
Thanks for your report, I have fixed it.
https://github.com/OsuSync/RealTimePPDisplayer/releases/tag/v1.6.9

**Comment by Ztumpie905509** [comment]
The issue was fixed in version 1.6.9, but still remains in 1.6.10
Please update this.
